src/View/AbstractDirectoryView.vala

```diff
@@ -1957,7 +1957,7 @@ namespace FM {
 
             var bookmark_menuitem = new Gtk.MenuItem ();
             bookmark_menuitem.add (new Granite.AccelLabel (
-                _("Add to Favorites"),
+                _("Add to Bookmarks"),
```

Comment:
So this used to just be "Bookmark". I think "Add to Bookmarks" is probably clearer (especially when translating), but I know either could work.